From: ?
Upstream:  xen-unstable

SIGKILL causes atexit to not be called, so mouse SDL cleanup wasn't
performed, so mouse wasn't released.


Index: xen-unstable/tools/python/xen/xend/image.py
===================================================================
--- xen-unstable.orig/tools/python/xen/xend/image.py
+++ xen-unstable/tools/python/xen/xend/image.py
@@ -412,7 +412,7 @@ class HVMImageHandler(ImageHandler):
         import signal
         if not self.pid:
             return
-        os.kill(self.pid, signal.SIGKILL)
+        os.kill(self.pid, signal.SIGTERM)
         os.waitpid(self.pid, 0)
         self.pid = 0
 
